tools/c7n_azure/c7n_azure/container_host/host.py

```diff
@@ -233,18 +282,17 @@ def update_periodic(self, policy):
         Update scheduled policies using cron type
         periodic scheduling.
         """
-        if policy.data.get('mode', {}).get('schedule'):
-            trigger = CronTrigger.from_crontab(policy.data['mode']['schedule'])
-            trigger.jitter = jitter_seconds
-            self.scheduler.add_job(self.run_policy,
-                                   trigger,
-                                   id=policy.name,
-                                   name=policy.name,
-                                   args=[policy, None, None],
-                                   coalesce=True,
-                                   max_instances=1,
-                                   replace_existing=True,
-                                   misfire_grace_time=20)
+        trigger = CronTrigger.from_crontab(policy.data['mode']['schedule'])
+        trigger.jitter = jitter_seconds
+        self.scheduler.add_job(self.run_policy,
+                               trigger,
+                               id=policy.name,
+                               name=policy.name,
+                               args=[policy, None, None],
+                               coalesce=True,
+                               max_instances=1,
+                               replace_existing=True,
+                               misfire_grace_time=20)
```

Comment:
Unrelated to this PR, but an idea: In the future we should try to make misfire_grace_time a percentage (~15%) of the trigger interval (maybe the trigger object has some property to give us a clue from the crontab).    If you have all your policies set to run at midnight and they are 20 seconds late then we wouldn't want to skip them, but if they are set to run every minute then for sure we should skip and wait for the next run.